.circleci/config.yml

```diff
@@ -137,6 +137,29 @@ jobs:
             curl -Os https://uploader.codecov.io/latest/linux/codecov
             chmod +x codecov
             ./codecov -t ${CODECOV_TOKEN} -Z -F "$REACT_VERSION-jsdom"
+
+  test_unit_vitest:
+    <<: *default-job
+    steps:
+      - checkout
+      - install_js
+      - run:
+          name: Tests fake browser (vitest)
+          command: pnpm vitest --project "jsdom/*" --coverage
+      - run:
+          name: Check coverage generated
+          command: |
+            if ! [[ -s coverage/lcov.info ]]
+            then
+              exit 1
+            fi
```

Comment:
@Janpot is it possible to test circleci steps without merging? 🤔

Reply:
Should be yes, when you add it to the workflow: https://github.com/mui/mui-x/blob/98013ae364f134b5d9e53dc8addf6e24c96e0e10/.circleci/config.yml#L332